### ISSUE TITLE
Retain_Graph=False also works

### ISSUE THREAD

**Issue by MR-TLL** [BlackDChase/Profit-Optimizer-A3C#15]
In loss.backward() , not specifying retain_graph=True , also works.
This looks logically correct to me , but will have to verify its correctness by running on a real env.
